src/editor/components/components/AddLayerPanel/createLayerFunctions.js

```diff
@@ -149,11 +149,22 @@ function createPrimitiveGeometry() {
   Events.emit('entitycreated', newEl);
 }
 
+function createIntersection() {
```

Comment:
Do you know you can just use `export function createIntersection` here and for all the functions you want to export instead of having an export { ... } at the end of the file?
You don't need to modify that in this PR, I'm just asking if it was a stylistic choice in the code base or if there was any other reason I'm not aware of.

Reply:
I stuck to the style that already is. Initially, I don’t know what reasons the programmer had to use this style:)